.pylintrc

```diff
@@ -68,7 +68,8 @@ disable=missing-docstring,
       ungrouped-imports, # Leave this up to isort
       wrong-import-order, # Leave this up to isort
       bad-continuation, # Leave this up to black
-      line-too-long # Leave this up to black
+      line-too-long, # Leave this up to black
```

Comment:
Such a bikeshedding-intensive change in the same PR?  Sneaky. 😃

Reply:
I think you're accusing me of adding a trailing comma to the last item in the list here? But I added a new item below! And that one doesn't have a trailing comma.

Reply:
Whoops, that's what happens when one is lazy when reading and the comma isn't highlighted.